### ISSUE TITLE
[cli] Rancher CLI limits apps listing to 1000 items

### ISSUE THREAD

**Issue by fabiorauber** [rancher/rancher#22177]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
Create more than 1000 apps in a Project. List them with the command ```rancher apps list```

**Result:**
The Rancher CLI lists the first 1000 apps, ordered by name.

**Other details that may be helpful:**

**Environment information**
- Rancher version: v2.2.7
- Rancher CLI version: v2.2.0
- Installation option (single install/HA): Single install

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Hosted, docker-machine-xenserver
- Machine type (cloud/VM/metal): VM
- Kubernetes version (use `kubectl version`): 
```
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.5", GitCommit:"2166946f41b36dea2c4626f90a77706f426cdea2", GitTreeState:"clean", BuildDate:"2019-03-25T15:19:22Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by fabiorauber** [comment]
Still seeing it in Rancher version 2.3.2.

**Comment by fabiorauber** [comment]
Still seeing it in Rancher v2.3.5

**Comment by fabiorauber** [comment]
Probably because of https://github.com/rancher/norman/blob/acbc1114636d7ef6390df2383da41cd8957f44a8/parse/collection.go#L12

**Comment by dramich** [comment]
@fabiorauber Thanks for the fix!

**Comment by dramich** [comment]
Available in v2.4.0-rc3

**Comment by jiaqiluo** [comment]
Since the bug is on the base function of the listing operation, we can validate the fix by listing any supported resource via the CLI.

Steps:
- add 1000+  namespaces to a project
- list namespaces in the project via Rancher CLI
<img width="1412" alt="Screen Shot 2020-02-05 at 4 26 06 PM" src="https://user-images.githubusercontent.com/6218999/73892984-90eae080-4835-11ea-9d07-c8a9c693eb4f.png">



The bug is reproduced with Rancher CLI  v2.3.2
```
➜  ./rancher -v
rancher version v2.3.2
➜  ./rancher namespace -q | wc -l
995
```

The fix is validated with Rancher CLI  v2.4.0-rc3
```
➜ ./rancher -v
rancher version v2.4.0-rc3
➜ ./rancher namespace -q | wc -l
1051
```
